examples/specs/layer_line_multi_value_tooltip.vl.json

```diff
@@ -0,0 +1,127 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
```

Comment:
Add description to credit @jakevdp that this is forked / modified from his block? (And remove that block from the "community" example so it's less redundant.)

cc: @jakevdp  I hope that crediting you in the description here is fine.

Reply:
Done!